cli/src/pcluster/constants.py

```diff
@@ -40,6 +40,7 @@
     "alinux2": {"user": "ec2-user", "root-device": "/dev/xvda"},
     "ubuntu1804": {"user": "ubuntu", "root-device": "/dev/sda1"},
     "ubuntu2004": {"user": "ubuntu", "root-device": "/dev/sda1"},
+    "rhel8": {"user": "ec2-user", "root-device": "/dev/sda1"},
```

Comment:
**Question (non-blocking):** is this taken from the redhat8 AMI after the build image?

Reply:
Originally I took it from the official RH8 image, but I checked it's the same also on the AMI after the build image